src/statespace/dart/WeldJoint.cpp

```diff
@@ -0,0 +1,32 @@
+#include <aikido/statespace/dart/WeldJoint.hpp>
+
+namespace aikido {
+namespace statespace {
+namespace dart {
+
+//=============================================================================
+WeldJoint::WeldJoint(
+      ::dart::dynamics::WeldJoint* _joint)
+  : Rn(_joint->getNumDofs())
+  , JointStateSpace(_joint)
+{
+}
+
+//=============================================================================
+void WeldJoint::convertPositionsToState(
+  const Eigen::VectorXd& _positions, StateSpace::State* _state) const
+{
+  setValue(static_cast<State*>(_state), _positions);
+}
+
+//=============================================================================
+void WeldJoint::convertStateToPositions(
+  const StateSpace::State* _state, Eigen::VectorXd& _positions) const
+{
+  _positions = getValue(static_cast<const State*>(_state));
```

Comment:
Nothing to get values from `_state`, which is zero dimensional. It would be sufficient to resize `_positions` to zero as `_positions.resize(0);` (see above)